guides/micronaut-data-jdbc-repository/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,4 @@
+datasources:
+  default:
+    url: jdbc:tc:mysql:8:///db
+    driverClassName: org.testcontainers.jdbc.ContainerDatabaseDriver
```

Comment:
On the one hand, this is the easiest thing to setup. On the other hand, it would make every test slow. Since a container will be started even if the user executes a test which does not use persistence.

Reply:
Not sure what the alternative would be. Convert the test to use H2 or mock the repository?